crates/net/network/src/session/handle.rs

```diff
@@ -46,32 +46,32 @@ impl PendingSessionHandle {
 /// be performed: chain synchronization, block propagation and transaction exchange.
 #[derive(Debug)]
 #[allow(unused)]
-pub(crate) struct ActiveSessionHandle {
+pub struct ActiveSessionHandle {
     /// The direction of the session
-    pub(crate) direction: Direction,
+    pub direction: Direction,
     /// The assigned id for this session
-    pub(crate) session_id: SessionId,
+    pub session_id: SessionId,
     /// negotiated eth version
-    pub(crate) version: EthVersion,
+    pub version: EthVersion,
     /// The identifier of the remote peer
-    pub(crate) remote_id: PeerId,
+    pub remote_id: PeerId,
     /// The timestamp when the session has been established.
-    pub(crate) established: Instant,
+    pub established: Instant,
     /// Announced capabilities of the peer.
-    pub(crate) capabilities: Arc<Capabilities>,
+    pub capabilities: Arc<Capabilities>,
     /// Sender half of the command channel used send commands _to_ the spawned session
-    pub(crate) commands_to_session: mpsc::Sender<SessionCommand>,
+    pub commands_to_session: mpsc::Sender<SessionCommand>,
     /// The client's name and version
-    pub(crate) client_version: Arc<String>,
+    pub client_version: Arc<String>,
     /// The address we're connected to
-    pub(crate) remote_addr: SocketAddr,
+    pub remote_addr: SocketAddr,
```

Comment:
I'm skeptical that we want them to be mutable,

I'd rather have getters for all of these instead even if more effort

Reply:
makes sense, added getters